3.0.0/README.md

```diff
@@ -0,0 +1,327 @@
+
+```JSON
+{
+  "data": [
+    "http://www.example.com/admin/data.geojson"
+  ]
+}
+```
+
+## 3.8 `description`
+
+OPTIONAL. Default: null.
+
+A text description of the set of tiles. The description can contain any valid unicode character as described by the JSON specification [RFC 8259](https://tools.ietf.org/html/rfc8259).
+
+```JSON
+{
+  "description": "We are volcanoes. When we women offer our experience as our truth, as human truth, all the maps change. There are new mountains."
+}
+```
+
+## 3.9 `fillzoom`
+
+OPTIONAL. Default: null.
+
+An integer specifying the zoom level at which to generate overzoomed tiles from. Implementations may generate overzoomed tiles from parent tiles if the requested zoom level does not exist. In most cases, overzoomed tiles are generated from the maximum zoom level of the set of tiles. If fillzoom is specified, the overzoomed tile is generated from the fillzoom level.
+
+For example, in a set of tiles with maxzoom 10 and _no_ fillzoom specified, if a request for a z11 tile comes through, the implementation will use the maximum z10 parent tiles to generate the new, overzoomed z11 tile. If the same TileJSON object had fillzoom specified at z7, a request for a z11 tile would use the z7 tile instead of z10.
+
+```JSON
+{
+  "fillzoom": 7
+}
+```
+
+## 3.10 `grids`
+
+OPTIONAL. Default: [].
+
+An array of interactivity endpoints. {z}, {x} and {y}, if present, are replaced with the corresponding integers. If multiple endpoints are specified, clients may use any combination of endpoints. All endpoints MUST return the same content for the same URL. If the array doesn't contain any entries, UTF-Grid interactivity is not supported for this set of tiles. See https://github.com/mapbox/utfgrid-spec/tree/master/1.2 for the interactivity specification.
+
+*Note: UTF-Grid interactivity predates GL-based map rendering and interaction. Map interactivity is now generally defined outside of the TileJSON specification and is dependent on the tile rendering library's features.*
+
+```JSON
+{
+  "grids": [
+    "http://www.example.com/earthsea/1.0.0/{z}/{x}/{y}.grid.json"
+  ]
+}
+```
+
+## 3.11 `legend`
+
+OPTIONAL. Default: null.
+
+Contains a legend to be displayed with the map. Implementations MAY decide to treat this as HTML or literal text. For security reasons, make absolutely sure that this field can't be abused as a vector for XSS or beacon tracking.
+
+```JSON
+{
+  "legend": "Dangerous zones are red, safe zones are green"
+}
+```
+
+## 3.12 `maxzoom`
+
+OPTIONAL. Default: 30. >= 0, <= 30.
+
+An integer specifying the maximum zoom level. MUST be >= minzoom.
```

Comment:
is it worth mentioning overzooming in this description?

Reply:
@samanpwbb what context do you think we should mention this? Similar to `fillzoom` above, I'm having a hard time determining how prescriptive or descriptive to be when it comes to how servers should act. I don't want to force tile servers to do anything they don't want to, but we can definitely provide some extra context/clarity for complex topics like overzooming. What about a descriptive helper sentence like: 

> Overzooming (link to a definition somewhere?) support is implemented by clients or servers. The specification does not enforce whether overzooming is available or not, merely provides information so servers and clients may better generate overzoomed tiles.

This feels like a statement that could exist outside fo the maxzoom field, too.

Reply:
After thinking about this more (from prompt by @stevage), what I'd really like to see is an explicit description somewhere in tileJSON about the server-side over and under-zoom range (see https://github.com/mapbox/tilejson-spec/pull/36#discussion_r213518662). 

This matters because often client applications (like Studio) are responsible for communicating to users who may not understand how tiles work why their data layers are or are not visible on the map at any given time.

Reply:
Interesting @samanpwbb - so you're thinking something like an overzoom property? Like:

```
{
  "minzoom": 0,
  "maxzoom": 15,
  "fillzoom": 8,
  "overzoom": 30
}
```

And this way the client knows it can safely make requests beyond the maxzoom (where data is available) but expect anything greater than maxzoom to be overzoomed. 

Would you mind opening a ticket to discuss for v4?